bitcoin/src/blockdata/transaction.rs

```diff
@@ -846,14 +851,15 @@ impl Transaction {
         Ok(())
     }
 
-    /// Is this a coin base transaction?
+    /// Checks if this is a coin base transaction.
```

Comment:
Since one bad company hijacked the term it'd make sense to add explanation for newbies:

> The first transaction in the block, the one receiving the reward, is called coin base transaction.
> It is impossible to check if the transaction is first in the block, so this function checks the structure of the transaction instead - the previous output must be all-zeros (creates satoshis "out of thin air").

Reply:
Agreed. Also, I think "coinbase" should be one word, though I'd be fine either way.

Reply:
Ok, if this needs changing I'll do all of the suggestions in this PR, converting to draft till I've done them.

Reply:
> Agreed. Also, I think "coinbase" should be one word, though I'd be fine either way.

This implies the function name should be `is_coinbase`. Please concept ack/nack this and I'll add it to my todo list.

Reply:
We have `Block::coinbase` to argue in favour of this rename also.

Reply:
ACK coinbase instead of `coin_base`/"coin base". I thought it was one word before but seeing it as two words I assumed I misremembered. Turns out I remembered right and the code was wrong.